### DISCUSSION TITLE
Account Locked, Bills Piling Up: Desperate for a Solution!!

### DISCUSSION THREAD

**Discussion by teddy950315** [community/community#123559] [Other Feature Feedback, Questions, & Ideas]
Request to Cancel GitHub Copilot Subscription Due to Account Block

Hello, I have been unable to access my GitHub account due to a block, yet my subscription to GitHub Copilot continues to be charged monthly. Despite numerous attempts to contact support over the past two months, I have received no responses, and the third charge is imminent. This situation is frustrating as I can neither use the service nor cancel the ongoing payments. Please address this issue promptly and help me regain access to my account or at least stop further charges. Thank you.

**Comment by teddy950315** [comment]
this ID is my 2nd ID, the real one is another. But they do not answer my ticket. I have no choice but to create a new account in order to communicate.

**Comment by hafiz-muhammad** [comment]
You can try contacting whoever your financial service is that you use to block future charges while you see about getting your account issue possibly resolved. I'm not sure how that will impact your account though.
